### ISSUE TITLE
Optimized branch lengths in aBSREL are all 0.0000

### ISSUE THREAD

**Issue by shablak** [veg/hyphy#1318]
When I ran an aBSREL analysis, the output summary stated that every single optimized branch length = 0.0000. I find this biologically unlikely and also doesn't match what I find in RAxML with the tree. I think this error may be occurring because I submitted an input tree without branch lengths? Should I resubmit a job with a tree with branch lengths included? If I add the branch lengths, is it important that they're close to the ML optimum (e.g. estimated by RAxML) or is it okay to put in dummy branch lengths such as 1.0 for every branch? 

Sort of funny to see my squished-flat tree; it made me smile.

**Comment by spond** [comment]
Dear @shablak,

HyPhy optimizes all branch lengths; if you provide some in the Newick string, they will be used as initial guesses. It seems like there's a bug in the branch length output; can you send me the data so I can replicate and fix the issues? spond at temple edu

Best,
Sergei

**Comment by spond** [comment]
Dear @shablak,

Just to confirm: where are you seeing 0.0000? On the screen, or in `json`?

Best,
Sergei

**Comment by shablak** [comment]
Hi Sergei,

I see it both on the screen and in the json file.

Shauna

**Comment by spond** [comment]
Dear @shablak,

I just ran the analysis using 2.5.30, and it looks like the analysis correctly reports branch lengths (small for many branches but non-zero). The error appears to occur in vision.hyphy.org/absrel module, where the branches are shown as having zero-length. @stephenshank : can you take a look?

For example, the `json` file shows 
```
"NEOPELMA_CHRYSOCEPHALUM_1":{
       "Baseline MG94xREV":0.006272293140378703,
       "Baseline MG94xREV omega ratio":10000000000,
       "Corrected P-value":0.02232418490789506,
       "Full adaptive model":0.006260012839626902,
       "Full adaptive model (non-synonymous subs/site)":0.006260012839626902,
       "Full adaptive model (synonymous subs/site)":1e-10,
       "LRT":19.0321938699999,
       "Nucleotide GTR":0.006183807650038019,
       "Rate Distributions":        [
[10000000000, 1] 
        ],
       "Rate classes":1,
       "Uncorrected P-value":2.466760763303322e-05,
       "original name":"NEOPELMA_CHRYSOCEPHALUM_1"
      },
```
yet on the vision page, we see 

<img width="835" alt="image" src="https://user-images.githubusercontent.com/1018513/114075339-71a17600-9873-11eb-82ca-7c74f18de094.png">

You can try using this page https://observablehq.com/@spond/displaying-absrel-results-using-branch-censoring/2 to load the `.json` into to confirm.

Best,
Sergei

**Comment by shablak** [comment]
Hi Sergei,

This works fine for me--I can view the branch lengths from the json file
loaded into that page. My main concern was that the program was not
functioning if it was genuinely estimating branch lengths of 0, which is
evidently not the case.

Thanks so much!

Shauna
